### DISCUSSION TITLE
Stop Loss Rates

### DISCUSSION THREAD

**Discussion by sgoodwintq** [CMSgov/hospital-price-transparency#20] [Q&A] [answered]
I would like to submit a request for an additional example scenario to be added (they are very helpful). 

What is a compliant way that a hospital can display a Stop Loss rate that is triggered when the total claim charges exceed a certain threshold? E.g., "when total claim charges exceed $200,000.00, the entire claim will be reimbursed at 50% of charges. this rate will supersede any other contracted rate that would otherwise apply". 

Might this provision be reflected as its own entry in the MRF as 'Stop Loss Claims', or be added as an additional note or algorithm to all other payer-negotiated rates? Or both/either? Thank you.

**Comment by RidenourK** [comment]
@sgoodwintq Thank you for your interest, we are working on more examples and will notify you when they are posted.

**Comment by RidenourK** [comment, marked as answer]
@sgoodwintq. Thank you for your patience. 

Please refer to the updated CSV and JSON documentation on how to encode data for a new optional data element, “general_contract_provisions”, as well as the updated CSV and JSON examples which include an example of how to encode data in this scenario.
